onnxruntime/core/framework/op_kernel.cc

```diff
@@ -191,21 +191,4 @@ OrtValue* OpKernelContext::GetOutputMLValue(int index) {
   return execution_frame_->GetMutableNodeInputOrOutputMLValue(output_arg_index);
 }
 
-#ifdef ENABLE_TRAINING
-Status OpKernelContext::SetOutputMLValue(int index, const OrtValue& ort_value) {
```

Comment:
>SetOutputMLValue [](http://example.com/codeflow?start=24&length=16)

May need to sync with Wei-Sheng and Peng on how to support custom autograd.Function.

Reply:
I did syn with Peng (did also invite Wei-Sheng), though they can solve their problem with partial run but it might just be easier to use this API. My suggestion was to enhance this API to also perform device check.